dev/integration_tests/ios_host_app/FlutterUITests/FlutterUITests.m

```diff
@@ -0,0 +1,117 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import XCTest;
+
+@interface FlutterUITests : XCTestCase
```

Comment:
This is a rewrite of FlutterViewControllerTests.

Reply:
FWIW, EarlyGrey is about to publish a Flutter plugin.